mealplanner-ui/src/pages/MealPlans/DuplicateMealPlan.tsx

```diff
@@ -3,7 +3,7 @@ import { commitMutation } from "relay-runtime";
 import environment from "../../relay/environment";
 
 const duplicateMealPlanGQL = graphql`
-mutation DuplicateMealPlanMutation($connections: [ID!]!, $mealPlanId: BigInt!, $personId: BigInt) {
+mutation DuplicateMealPlanMutation($connections: [ID!]!, $mealPlanId: BigInt!) {
```

Comment:
You should leave the personId as the next line requires it

Reply:
Ah, this is an accident. I know when I was testing locally, it had the p_id. I must have made a mistake while commiting. I'll fix this right away.